src/cmds/qmgr.c

```diff
@@ -2004,8 +2004,13 @@ display(int otype, int ptype, char *oname, struct batch_status *status,
 				printf("Server %s\n", status->name);
 		}
 		else if (otype == MGR_OBJ_SCHED) {
-			if (!format)
-				printf("Sched %s\n", status->name);
+			if (format) {
+					printf("#\n# Create and define scheduler %s\n#\n", status->name);
+					printf("create sched %s\n", status->name);
+				}
+				else
+					printf("Scheduler %s\n", status->name);
```

Comment:
Since the default scheduler name has changed to "main"(as opposed to "1") , printing of scheduler objects information is not backward compatible anyways and hence we thought we will give the detailed information of the command used to create scheduler object while printing which is in line with the way queue information is printed.  We are also planning to add this to the documentation Please know your thoughts on the same.